packages/mobile/src/navigator/NavigatorWrapper.tsx

```diff
@@ -175,6 +181,9 @@ export const NavigatorWrapper = () => {
   return (
     <NavigationContainer
       ref={navigationRef}
+      onReady={() => {
```

Comment:
I think this is what is causing your linting error. You can either ignore it or define the function outside of the component properties